buildconfig/dev-packages/rpm/mantid-developer/mantid-developer.spec

```diff
@@ -71,8 +70,6 @@ Requires: qt5-qtx11extras
 Requires: qt5-qtx11extras-devel
 Requires: qt-devel >= 4.6
```

Comment:
Should we bump this to `> 5`?

Reply:
`4.8.7-9.el7_9` is the version we have on our RHEL7 machines. We could probably move to `>= 4.8`.

Reply:
Do we need it? Is it not the qt4 dependencies?

Reply:
I've never tried without it

Reply:
So it looks like we can remove `qt-devel`, `qtwebkit-devel` & `qscintilla-devel` and this should completely remove Qt4. I tested a build locally in a docker container and it worked.